setup.py

```diff
@@ -107,6 +107,7 @@
     "deepspeed>=0.9.3",
     "diffusers",
     "dill<0.3.5",
+    "einops",
```

Comment:
I don't think you're using it here, and even if it's the case, let's remove it! 
We're trying to keep minimum dependencies!

Reply:
ah, yep, I translated the einops code from the original repo to use torch